src/test/java/Problem2Test.java

```diff
@@ -8,7 +8,39 @@
     // Okay to change
     @Test
     public void testBubbleSort() {
-        assertTrue(false);
-        // add your tests here
+        int inputs[][] = {
+                {1, 2, 3, 4, 5},
+                {4, 3, 5, 2, 1},
+                {},
+                {1},
+                {1, 1, 1},
+                {4, 4, 2, 1, 2},
+                {5, 4, 3, 2, 1},
+                {-1, 3, -5},
+                {-2},
+                {-3, -2, -1, 0}
+        };
+        int answers[][] = {
+                {1, 2, 3, 4, 5},
+                {1, 2, 3, 4, 5},
+                {},
+                {1},
+                {1, 1, 1},
+                {1, 2, 2, 4, 4},
+                {1, 2, 3, 4, 5},
+                {-5, -1, 3},
+                {-2},
+                {-3, -2, -1, 0}
+        };
+        assertEquals(inputs.length, answers.length);
+        for (int i = 0; i < inputs.length; i++) {
+            Problem2.bubbleSort(inputs[i]);
+            try {
+                assertArrayEquals(inputs[i], answers[i]);
```

Comment:
the order of parameter is reversed. 

1st argument is the expected value, and 2nd argument is the actual value